src/Scope/FeatureEnum.php

```diff
@@ -46,6 +47,7 @@ class FeatureEnum extends Enum
         self::ADHERENT_FORMATIONS,
         self::COMMITTEE,
         self::GENERAL_MEETING_REPORTS,
+        self::DOCUMENTS,
```

Comment:
could you add a translation for this new feature [here](https://github.com/EnMarche/en-marche.fr/blob/9aa3d1f3465f19974ee8f14578b04f57740e2d54/translations/messages+intl-icu.fr.yml#L1356), please?